NEWS.md

```diff
@@ -1,3 +1,21 @@
+# admiral 0.12.0.9000
```

Comment:
this should have `admiral (development version)` instead of the number.  I think `usethis::use_version("dev")` will update for you

Reply:
Updated, although `use_version("dev")` did not work well (it increased to 0.12.0.9001 and added an additional heading in `NEWS.md` and the commit failed). I think we should call `use_version("dev")` once directly after each release.

Reply:
I believe every time we merge into `main` while `developing` this will be incremented.  @zdz2101 @ddsjoberg

Reply:
I would keep it simple and always stay at x.x.x.9000. Even in the "R packages" book it is suggested to increment the development version only in exceptional cases:
> Increment the development version, e.g. from 9000 to 9001, if you’ve added an important feature and you (or others) need to be able to detect or require the presence of this feature. For example, this can happen when two packages are developing in tandem. This is generally the only reason that we bother to increment the development version. This makes in-development versions special and, in some sense, degenerate. Since we don’t increment the development component with each Git commit, the same package version number is associated with many different states of the package source, in between releases.

Reply:
I agree it will make it a lot easier on us if we don't have to change this every time we merge into `main`.  Seems like this will cause a lot of tedious merge conflicts.

Reply:
This does seem to make more sense, I'll remove the running of the line from the PR template, and it can be added as a standard post-release task